module/VuFind/src/VuFind/Hierarchy/Driver/ConfigurationBased.php

```diff
@@ -131,4 +131,22 @@ public function getCollectionLinkType()
         return isset($this->config->Collections->link_type)
             ? ucwords(strtolower($this->config->Collections->link_type)) : 'All';
     }
+
+    /**
+     * Get the Solr field name used for grouping together collection contents
+     *
+     * @param bool $hasSearch Is the user performing a search?
+     *
+     * @return string
+     */
+    public function getCollectionField(bool $hasSearch): string
+    {
+        if ($hasSearch && null !== ($this->config->Collections->search_container_id_field ?? null)) {
```

Comment:
This isn't quite what I suggested. Is there a reason for not taking it into a variable right away?

Reply:
Sorry, didn't look closely enough at that diff. Should be better now!